plugins/rule_engines/irods_rule_language/src/configuration.cpp

```diff
@@ -501,7 +500,7 @@ int load_rules(const char* irbSet, const std::vector<std::string> &irbs, const i
                             }
                         }
                         catch (std::out_of_range& ex) {
-                            logger::rule_engine::warn("load_rules: attempted to access nonexistent rulebase [{}]", irb);
+                            log_re::warn("load_rules: attempted to access nonexistent rulebase [{}]", irb);
```

Comment:
Let's use `__func__` or `__FUNCTION__`, etc for capturing the function name in log messages. That helps keep the log message and function name in sync (in case of renames).